finetune/encoding/target_encoders.py

```diff
@@ -135,6 +136,45 @@ def transform(self, y):
     def inverse_transform(self, y):
         return [self.encoder.decode(y_i.tolist()) for y_i in y]
 
+class SequenceLabelingTextEncoder(Seq2SeqLabelEncoder):
+    def transform(self, y):
+        all_labels = []
+        for y_i in y:
+            # List of dictionaries, where the key is the label and the value is
+            # the span being labeled
+            labels = [{span["label"]: span["text"]} for span in y_i]
```

Comment:
Maybe note that there's an expectation these follow some sort of order for future interns to avoid potential sadness.